SyntaxError ?

Hi, I follow all the steps and when I would like to run the dashboard, I got these errors : 

> 
> /parse_server/parse-dashboard/Parse-Dashboard/index.js:18
>   .then(config => {
>                ^^
> SyntaxError: Unexpected token =>
>     at exports.runInThisContext (vm.js:73:16)
>     at Module._compile (module.js:443:25)
>     at Object.Module._extensions..js (module.js:478:10)
>     at Module.load (module.js:355:32)
>     at Function.Module._load (module.js:310:12)
>     at Function.Module.runMain (module.js:501:10)
>     at startup (node.js:129:16)
>     at node.js:814:3
> /parse_server/parse-dashboard/plugins/svg-prep.js:10
> const fs = require('fs');
> ^^^^^
> SyntaxError: Use of const in strict mode.
>     at exports.runInThisContext (vm.js:73:16)
>     at Module._compile (module.js:443:25)
>     at Object.Module._extensions..js (module.js:478:10)
>     at Module.load (module.js:355:32)
>     at Function.Module._load (module.js:310:12)
>     at Module.require (module.js:365:17)
>     at require (module.js:384:17)
>     at Object.<anonymous> (/Users/charlyberthet/Documents/_projets/butler/parse_server/parse-dashboard/base.config.js:11:21)
>     at Module._compile (module.js:460:26)
>     at Object.Module._extensions..js (module.js:478:10)


Could you explain me what I missed ?
Thanks

npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "run" "dashboard"
npm ERR! node v0.12.0
npm ERR! npm  v3.8.0
npm ERR! code ELIFECYCLE
npm ERR! @ dashboard: `node ./Parse-Dashboard/index.js & webpack --config build.config.js --progress --watch`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ dashboard script 'node ./Parse-Dashboard/index.js & webpack --config build.config.js --progress --watch'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./Parse-Dashboard/index.js & webpack --config build.config.js --progress --watch
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/charlyberthet/Documents/_projets/butler/parse_server/parse-dashboard/npm-debug.log

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "start": "node ./Parse-Dashboard/index.js"
   },
   "engines": {
-    "node": ">=4.3",
+    "node": ">=4.3"
   },
   "jest": {
     "testPathDirs": [

--- a/package.json
+++ b/package.json
@@ -44,6 +44,9 @@
     "prestart": "webpack --config build.config.js --progress",
     "start": "node ./Parse-Dashboard/index.js"
   },
+  "engines": {
+    "node": ">=4.3",
+  },
   "jest": {
     "testPathDirs": [
       "lib"

